js-basics/js-async-functions_relay-race/js/index.js

```diff
@@ -24,7 +24,11 @@ startRaceButton.addEventListener("click", () => {
    **/
 
   // --v-- write your code here --v--
-
+  (async function () {
+    await animateRunner(runner1);
+    await animateRunner(runner2);
+    await animateRunner(runner3);
+  })();
   // --^-- write your code here --^--
```

Comment:
I don't understand why you wrapped everything in another function ... Does it work? If so it is approved :)

Reply:
it works